perlin.py

```diff
@@ -46,7 +46,7 @@ def get_perlin_noise(x: float, y: float, base: float, scale: float = 0.005, octa
     :param octaves: the octaves to use when sampling, default is 3
     :return: a normalised float of the sample in noisemap
     """
-    return (noise.pnoise2(x=x * scale, y=y * scale, octaves=octaves, base=base) + 1) / 2
+    return (noise.pnoise2(x=x * scale, y=y * scale, octaves=octaves, base=int(base)) + 1) / 2
```

Comment:
Why cast base to int? Signature seems to allow for floats.

Reply:
Hang on. You told me base should be an int, when I said the noise function didn't work due to an "expected int, got float" error. And it also fixed the problem.

Reply:
Just tested it. Can confirm that base has to be an integer.

Reply:
Hm, I must've forgotten that. But cool, then this change just also has to be applied to #14.

Reply:
I changed the type hints for our functions to be integers as well. See 7c1b745feda92